Redis connection error

Trying the test out authelia following the docs here:

https://docs.authelia.com/deployment/deployment-lite.html

It simply says to modify the configuration.yml and docker-compose.yml with your respective domains and secrets. 

Trying to run authelia and I get the following error:

panic: Redis connection error: ERR AUTH <password> called without any password configured for the default user. Are you sure your configuration is correct?

I would think the default config would have authelia configured withe a default password. What am I missing here. The docs need to be a bit more clear.

--- a/compose/lite/configuration.yml
+++ b/compose/lite/configuration.yml
@@ -44,7 +44,7 @@ session:
     host: redis
     port: 6379
     # This secret can also be set using the env variables AUTHELIA_SESSION_REDIS_PASSWORD_FILE
-    password: authelia
+    # password: authelia
 
 regulation:
   max_retries: 3

